modules/browser/detect_lastpass/command.js

```diff
@@ -4,26 +4,41 @@
 // See the file 'doc/COPYING' for copying permission
 //
 
+$j = jQuery.noConflict();
```

Comment:
Is this necessary? Why is it outside of the `beef.execute` call?

Reply:
No, not necessary. Good pickup. Will address